apps/prairielearn/src/pages/home/home.js

```diff
@@ -9,7 +9,7 @@ var sql = sqldb.loadSqlEquiv(__filename);
 
 router.get('/', function (req, res, next) {
   res.locals.navPage = 'home';
-  res.locals.isAuthenticated = !!res.locals.authn_user;
+  res.locals.isAuthenticated = !!res.locals.authn_user && true;
```

Comment:
I used this as a way to test the unauthenticated page locally (just with `&& false` instead), as there's otherwise no way to view a page without being forcefully authenticated.

Reply:
I see you removed it later anyway.